orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/AccountInterventionService.java

```diff
@@ -180,13 +180,18 @@ private AccountIntervention retrieveAccountIntervention(
 
     private HttpResponse<String> sendRequestToAis(HttpRequest request) {
         HttpResponse<String> httpResponse = null;
+        var start = System.nanoTime();
         try {
             httpResponse = httpClient.send(request, HttpResponse.BodyHandlers.ofString());
 
         } catch (IOException | InterruptedException e) {
             logAndThrowAccountInterventionException(
                     "Failed to send request to Account Intervention Service.");
         }
+        var durationMs = (System.nanoTime() - start) / 1000L;
```

Comment:
Use `currentTimeMillis()` instead of converting?

Reply:
Ah I think this will end up in microseconds not milliseconds

Reply:
was that the intention?

Reply:
See https://stackoverflow.com/questions/1770010/how-do-i-measure-time-elapsed-in-java/1776053#1776053